components/ActionSheet/examples/ActionSheetComparisonExampleViewController.m

```diff
@@ -27,61 +30,48 @@ @interface ActionSheetComparisonExampleViewController ()
 
 @end
 
-@implementation ActionSheetComparisonExampleViewController {
-  MDCButtonScheme *_buttonScheme;
-}
+@implementation ActionSheetComparisonExampleViewController
 
 - (instancetype)init {
   self = [super init];
   if (self) {
     self.title = @"Action Sheet";
-    _colorScheme = [[MDCSemanticColorScheme alloc] init];
-    _typographyScheme = [[MDCTypographyScheme alloc] init];
+    _containerScheme = [[MDCContainerScheme alloc] init];
```

Comment:
I can't add a comment up there, but please change the ContainerScheme type to `id<MDCContainerScheming>`

Reply:
Fixed